sllm/serve/routers/roundrobin_router.py

```diff
@@ -324,6 +330,13 @@ async def _finish_instance(self, instance_id: str):
                 logger.error(f"Instance {instance_id} not found")
                 return
             instance = self.deleting_instances.pop(instance_id)
+
+        # ensure there's no requests being processed
+        while instance.queue_length > 0:
```

Comment:
I was wondering if this check is necessary since `instance.backend_instance.stop` already ensures that all current requests are completed, as shown here:  
https://github.com/ServerlessLLM/ServerlessLLM/blob/437dd374c2a04da680d4c2764779a8554582052b/sllm/serve/backends/transformers_backend.py#L316-L326